vyxal/elements.py

```diff
@@ -796,11 +800,14 @@ def dyadic_minimum(lhs, rhs, ctx):
 def e_digits(lhs, ctx):
     """Element ∆Ė
     (int) -> e_digits(a)
+    (str) -> evaluate as sympy
     """
     if vy_type(lhs) == NUMBER_TYPE:
-        estr = str(sympy.N(sympy.E, int(lhs) + 1))
+        estr = str(sympy.N(sympy.E, int(lhs) + 2))
```

Comment:
What was this for btw? (this is a comment, not a request for a change, feel free to resolve it)

Reply:
Because it was returning `3` for `0`